### PR TITLE
RIA-8797 - Set listing location and whether hearing is remote when system triggers "Edit case listing"

### PR DESCRIPTION
**Pull request by robin-ieong**
…stem triggers Edit case listing

**Before creating a pull request make sure that:**

- [x ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-8797

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/ListedHearingService.java [hunk 1]
```diff
@@ -277,12 +277,12 @@ private HearingCentre getHearingCentre(ServiceData serviceData) {
             : HearingCentre.getHearingCentreByEpimsId(getHearingVenueId(serviceData));
     }
 
-    private boolean isRemoteHearing(ServiceData serviceData) {
+    public boolean isRemoteHearing(ServiceData serviceData) {
         final String nextHearingChannel = getHearingChannels(serviceData).get(0).name();
```
Comment (rajeshthuraiyur) [inline src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/ListedHearingService.java:281]: no unit test? it is now public method, it is better to have a unit test at least for test coverage

File: src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/ListedHearingService.java [hunk 2]
```diff
@@ -277,12 +277,12 @@ private HearingCentre getHearingCentre(ServiceData serviceData) {
...
         return nextHearingChannel.equals(VID.name()) || nextHearingChannel.equals(TEL.name());
     }
 
-    private String getHearingCourtName(ServiceData serviceData, List<CourtVenue> courtVenues) {
```
Comment (rajeshthuraiyur) [inline src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/ListedHearingService.java:285]: can we add an unit test?

**Review by augustio — APPROVED** [review]
LGTM.

File: src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/ListedHearingService.java (hunk 2 above)
Comment (robin-ieong) [reply src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/ListedHearingService.java:285]: added unit test

**Pull request merged**